libnetwork/drivers/bridge/setup_device_linux_test.go

```diff
@@ -92,3 +92,25 @@ func TestGenerateRandomMAC(t *testing.T) {
 		t.Fatalf("Generated twice the same MAC address %v", mac1)
 	}
 }
+
+// Regression test for https://github.com/moby/moby/issues/47308
+func TestInvalidMTU(t *testing.T) {
+	defer netnsutils.SetupTestOSContext(t)()
+
+	nh, err := netlink.NewHandle()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer nh.Close()
+
+	config := &networkConfiguration{BridgeName: DefaultBridgeName, Mtu: 100000}
```

Comment:
```suggestion
	config := &networkConfiguration{BridgeName: DefaultBridgeName, Mtu: 9000}
```

Reply:
Just realized your previous suggestion meant we'd be unable to write a proper regression test for https://github.com/moby/moby/issues/47308. But I prefer to keep that more selective condition as my comment is already explicit enough to hardly break this fix unintentionally.

Reply:
I was thinking more about failing fast if a user sets an absurd MTU, and future-proofing against a custom kernel which accepts MTU >  65535 for some reason. But we didn't have fail-fast behaviour before, so it's not a big deal if we don't do it (while EL7 is supported)

Reply:
Mock the netlink handle?